pkg/kubelet/prober/worker.go

```diff
@@ -204,6 +205,15 @@ func (w *worker) doProbe() (keepGoing bool) {
 		return true
 	}
 
+	// event message to be logged stating that the probe is healthy again and the amount of time the pod was unhealthy.
+	if w.probeType == readiness && w.lastResult == results.Failure && result == results.Success {
+		timeUnhealthy := time.Duration(w.spec.PeriodSeconds*int32(w.resultRun+1)) * time.Second
+		ref, hasRef := w.probeManager.prober.refManager.GetRef(w.containerID)
+		if hasRef {
+			w.probeManager.prober.recorder.Eventf(ref, v1.EventTypeNormal, events.ContainerTurnHealthy, "Readiness probe turn healthy again and the time of unhealthy is : %v", timeUnhealthy)
```

Comment:
`"Readiness probe is now healthy again after failing for %v"`

Slight change in error message. What do you think?